### ISSUE TITLE
Remove Time-traveling warning

### ISSUE THREAD

**Issue by malixsys** [postcss/autoprefixer#1318]
Is this necessary? In `dev` there may well be no targets and the warnings will needlessly show up...
https://github.com/postcss/autoprefixer/blob/0283e319ff7bcde0454c959036e0d91ae0bb326f/lib/autoprefixer.js#L43

**Comment by Timer** [comment]
Furthermore, just because a build doesn't require prefixing in the present doesn't mean that a few weeks in the future wouldn't introduce new properties that need prefixing. It'd be a shame to remove `autoprefixer` config in this case.

**Comment by ai** [comment]
> Furthermore, just because a build doesn't require prefixing in the present doesn't mean that a few weeks in the future wouldn't introduce new properties that need prefixing.

Browsers stopped adding new vendor prefixes. It is very unlikely that we will have any new vendor prefixes in the future.

>  In dev there may well be no targets and the warnings will needlessly show up...

What Browserslist config do you have?

**Comment by malixsys** [comment]
```json
"browserslist": {
    "production": [
      "last 2 iOS major versions",
      "ie 11"
    ],
    "development": [
      "last 1 chrome major versions"
    ]
}
```

**Comment by ai** [comment]
Strange. You should have prefixes for both environments.

Can you debug the code to find why do you have empty prefixes list?

**Comment by malixsys** [comment]
What prefixes should be found for `last 1 chrome major versions`?

**Comment by ai** [comment]
@malixsys you can check it by running `npx autoprefixer --info`

```
➜ BROWSERSLIST="last 1 chrome major versions" npx autoprefixer --info
Browsers:
  Chrome: 81

These browsers account for 8.71% of all users globally

Properties:
  appearance: webkit
  background-clip: webkit
  box-decoration-break: webkit
  color-adjust: webkit
  mask-border-outset: webkit
  mask-border-repeat: webkit
  mask-border-slice: webkit
  mask-border-source: webkit
  mask-border-width: webkit
  mask-border: webkit
  mask-clip: webkit
  mask-composite: webkit
  mask-image: webkit
  mask-origin: webkit
  mask-position: webkit
  mask-repeat: webkit
  mask-size: webkit
  mask: webkit
  text-emphasis-color: webkit
  text-emphasis-position: webkit
  text-emphasis-style: webkit
  text-emphasis: webkit

Values:
  cross-fade: webkit
  fill-available: webkit
  fill: webkit
  image-set: webkit
  stretch: webkit
```

**Comment by malixsys** [comment]
```
npx: installed 25 in 3.173s
Browsers:
  Chrome: 83

These browsers account for 4.07% of all users globally

Properties:
  appearance: webkit
  background-clip: webkit
  box-decoration-break: webkit
  color-adjust: webkit
  mask-border-outset: webkit
  mask-border-repeat: webkit
  mask-border-slice: webkit
  mask-border-source: webkit
  mask-border-width: webkit
  mask-border: webkit
  mask-clip: webkit
  mask-composite: webkit
  mask-image: webkit
  mask-origin: webkit
  mask-position: webkit
  mask-repeat: webkit
  mask-size: webkit
  mask: webkit
  text-emphasis-color: webkit
  text-emphasis-position: webkit
  text-emphasis-style: webkit
  text-emphasis: webkit

Values:
  cross-fade: webkit
  fill-available: webkit
  fill: webkit
  image-set: webkit
  stretch: webkit
```

**Comment by ai** [comment]
You have prefixes in the lost and should not see the warning. Seems like some tool use Autoprefixer wrong. Can you open Autoprefixer sources and put a error throwing before the warning so we will see stack trace?

**Comment by Pwntheon** [comment]
Had this show up today and i must say i got pretty ticked off. The warning gives very little information about why it suddenly showed up, what the problem is, how to fix it, and where to learn more.

Most hits on google are 1 or 0 votes stackoverflow answers that give no more context.

**Comment by ai** [comment]
What do you think about this text: c1b26d1 ?

**Comment by ai** [comment]
The fix was released in 10.2.6
